addons/stock/views/stock_picking_views.xml

```diff
@@ -126,6 +126,7 @@
             <field eval="12" name="priority"/>
             <field name="arch" type="xml">
                 <form string="Transfer">
+                    <field name="use_create_lots" invisible="1"/>
```

Comment:
I thought after automatic addition of invisible fields, we needed to have an XML comment explaining why it's necessary to have invisible="1" field, but runbot doesn't seem to cry about it (but you could still add it)

the tabulation is wrong for the lines below, but tabulating this one looks odd, I would remove it

```suggestion
                <field name="use_create_lots" invisible="1"/> <!-- used by embedded account.move view -->
```